java/test/jmri/jmrix/cmri/serial/cmrimetrics/CMRInetMetricsCollectorTest.java

```diff
@@ -0,0 +1,45 @@
+package jmri.jmrix.cmri.serial.cmrimetrics;
+
+import apps.tests.Log4JFixture;
+import jmri.util.JUnitUtil;
+import jmri.jmrix.cmri.CMRISystemConnectionMemo;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+import java.awt.GraphicsEnvironment;
+
+/**
+ * Test simple functioning of CMRInetMetricsCollector
+ *
+ * @author	Chuck Catania Copyright (C) 2017
+ */
+public class CMRInetMetricsCollectorTest {
+
+    @Test
+    public void testStringCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        CMRInetMetricsCollectorTest action = new CMRInetMetricsCollectorTest(); 
+        Assert.assertNotNull("exists", action);
+    }
+
+    @Test
+    public void testCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        CMRInetMetricsCollectorTest action = new CMRInetMetricsCollectorTest(); 
+        Assert.assertNotNull("exists", action);
+    }
+
+    @Before
+    public void setUp() {
+        Log4JFixture.setUp();
```

Comment:
Please replace Log4JFixture.setUp() with JUnitUtil.setUp() and remove the following line.

Reply:
Replaced and removed line.